src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1461 @@
+                 ut_control_GetMapString(bool_mapTable, isarcport),iport);
+    }
+
+exit:
+UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get edid.
+*
+* This test function is to get edid bytes of  selected HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 009@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_get_edid(void)
+{
+    gTestID = 9;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    int length = 0;
+    unsigned char edidbytes[255] = {0};
+    int select;
+    dsHdmiInPort_t port = dsHDMI_IN_PORT_NONE;
+    uint8_t numInputPorts = 0;
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+    printf("\n**********Please select port***********\n");
+    scanf("%d", &select);
+    readAndDiscardRestOfLine(stdin);
+
+    if(select <= 0 || select >numInputPorts)
+    {
+      printf("\n invalid port selected\n");
+      goto exit;
+    }
+
+    port = select;
+    ret = dsGetEDIDBytesInfo(port, edidbytes, &length);
+    if(ret != dsERR_NONE)
+    {
+       UT_LOG_ERROR("Result: Failed dsGetEDIDBytesInfo Error:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    } else {
+           UT_LOG_INFO("Result : Passed dsGetEDIDBytesInfo port type: %s, \n portnumber: %d,\n\t"
+                         "edidinfo: %s", ut_control_GetMapString(dsHdmiInPort_mapTable, port), port, edidbytes);
+    }
+
+exit:
+UT_LOG_INFO("Out %s", __FUNCTION__);
+}
+
+/**
+* @brief This test to get spdinfo.
+*
+* This test function is to get spdinfo of  selected HdmiInput port on platform.
+*
+* **Test Group ID:** 03@n
+* **Test Case ID:** 010@n
+*
+* **Test Procedure:**
+* Refer to Test specification documentation
+* [ds-hdmi-in_halSpec.md](../../docs/pages/ds-hdmi-in_halSpec.md)
+*/
+void test_l3_HdmiIn_spd_info(void)
+{
+    gTestID = 10;
+    UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t ret   = dsERR_NONE;
+    unsigned char spdinfo[sizeof(struct dsSpd_infoframe_st)] = {0};
+    dsHdmiInPort_t port = dsHDMI_IN_PORT_NONE;
+    int select;
+    uint8_t numInputPorts = 0;
+    
+
+    numInputPorts = UT_KVP_PROFILE_GET_UINT8("dsHdmiIn/numberOfPorts");
+    printf("\n**********Please select port***********\n");
+    scanf("%d", &select);
+    readAndDiscardRestOfLine(stdin);
+
+    if(select <= 0 || select >numInputPorts)
+    {
+      printf("\n invalid port selected\n");
+      goto exit;
+    }
+   
+    port = select;
+   
+    ret = dsGetHDMISPDInfo(port, spdinfo); 
+    if(ret != dsERR_NONE)
+    {
+       UT_LOG_ERROR("Result: Failed dsGetHDMISPDInfo Error:[%s]",ut_control_GetMapString(dsError_mapTable, ret));
+    } else {
+          /* UT_LOG_INFO("Result : Passed dsGetHDMISPDInfo porttype: %s, portnumber: %s,\t"
```

Comment:
remove commented out code...It's either expected to be there or it should be removed.

Why would you want to hide this data from view?

Reply:
removed .